### PR TITLE
feat: Delete attribute modal

### LINKED ISSUE #17543: [CAL-4687] Deleting attributes should have a required confirmation
Deleting an attribute from an organization should have a confirmation modal and also state that it will be un assigning all users who have had this attribute assigned to them. 

### Steps to Reproduce

1. Log in as an org (see Prisma/seed.ts)
2. Goto settings 
3. Goto attributes 
4. Create an attribute 
5. Delete an attribute 
6. Notice no confirmation mod

<sub>[CAL-4687](https://linear.app/calcom/issue/CAL-4687/deleting-attributes-should-have-a-required-confirmation)</sub>

### PR DESCRIPTION
**Pull request by Amit91848**
## What does this PR do?

- Fixes #17543 
- Fixes CAL-4687

https://github.com/user-attachments/assets/70416a94-2118-4672-b408-7ae6a0d59b97

## Mandatory Tasks (DO NOT REMOVE)

- [x] I have self-reviewed the code (A decent size PR without self-review might be rejected).
- [x] I have updated the developer docs in /docs if this PR makes changes that would require a [documentation change](https://cal.com/docs). If N/A, write N/A here and check the checkbox.
- [x] I confirm automated tests are in place that prove my fix is effective or that my feature works.

### CODE REVIEW

File: packages/features/ee/organizations/pages/components/DeleteAttributeModal.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import type { RouterOutputs } from "@calcom/trpc/react";
+import { trpc } from "@calcom/trpc/react";
+import { Dialog, ConfirmationDialogContent, showToast } from "@calcom/ui";
+
+type AttributeItemProps = RouterOutputs["viewer"]["attributes"]["list"][number];
+
+export function DeleteAttributeModal({
+  attributeToDelete,
+  setAttributeToDelete,
```
Comment (anikdhabal) [inline packages/features/ee/organizations/pages/components/DeleteAttributeModal.tsx:12]: Let's move this inside /attributes

**Review by Praashh — CHANGES_REQUESTED** [review]
hey @Amit91848 , can you move the `modal` to `attributes` folder

**Comment by Amit91848** [comment]
> hey @Amit91848 , can you move the `modal` to `attributes` folder

Done

**Review by sean-brydon — APPROVED** [review]
LGTM! Thanks for tackling this.

**Review by Praashh — APPROVED** [review]

**Pull request merged**